index.html

```diff
@@ -98,8 +104,10 @@ <h2 class="card-name">Tonic</h2>
       </ul>
       <button class="sec-button">see project</button>
     </div>
+    </div>
     <div class="card">
-      <img class="pic" src="img/Second pic.png" alt="second-img" />
+      <img class="pic pic2" src="img/desktop/Snapshoot Portfolio2.png" alt="second-img" />
+      <div class="Desktop-common">
       <h2 class="card-name">Multi-post stories</h2>
       <ul class="can-bak">
         <li class="cann">canopy</li>
```

Comment:
It looks great, but to follow the Figma template you can add some padding to the top and left for the content section.

![image](https://user-images.githubusercontent.com/34779928/194393299-7ea09faa-9cd9-4db9-968a-065e6560c8f7.png)